tests/cbmc/proofs/s2n_stuffer_write_vector_size/Makefile

```diff
@@ -40,6 +38,6 @@ REMOVE_FUNCTION_BODY += s2n_blob_slice
 REMOVE_FUNCTION_BODY += s2n_stuffer_resize
 REMOVE_FUNCTION_BODY += s2n_stuffer_wipe_n
 
- UNWINDSET += s2n_stuffer_write_network_order.4:$(shell echo $$((1 + $(MAX_LENGTH))))
+UNWINDSET += s2n_stuffer_write_network_order.4:9
```

Comment:
Again I'm a little surprised to see this hardcoded number.

Reply:
Implemented as suggested.